statistics/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -57,6 +57,8 @@ interface Pixel {
         const val OS_VERSION = "os_version"
         const val DEFAULT_BROWSER = "default_browser"
         const val EMAIL = "email"
+        const val MESSAGE_SHOWN = "message"
```

Comment:
all pixels that use this parameter (existing and new) are in the :app module, so I think it needs to be here

Reply:
but why, @nalcalag ? I don't think "because other pixels do that" is a strong reason to keep using the same patter if (assuming) is not a good one

Reply:
I think we should take a good look at some RMF classes. Some logic seems to be misplaced, like `RemoteMessagingModel` for instance.
I don't have the full context yet, but I don't see very compelling reason to put RMF implementation details in the :app module.

I'd agree it's out of scope for this PR though. We can do on a separate task, but should be done soonish

Reply:
Why can't it be moved into `remoteMessagingRepository.markAsShown` ?

Reply:
This is just a parameter that all remote messaging pixels have, which gives the ID of the message shown. Because we have existing pixels in the app module, even if we create the new ones in the remote-messaging module, they will need to be duplicated in order to access them from both places.

As I mentioned in my last comment, I believe this pixel refactor is beyond the scope of the project, and it would be better to address it as a separate task so we have consistency between all remote-messaging pixels.

Reply:
Captured this in https://app.asana.com/0/1200156640058969/1205201076152501/f

If I'm not wrong this can be quickly, but I'm ok if we don't do it here.